demo/combo-box-styling-demos.html

```diff
@@ -5,177 +5,92 @@
-    </p>
-    <vaadin-demo-snippet id="combo-box-styling-demos-custom-item-template" when-defined="vaadin-combo-box">
-      <template preserve-content>
-        <vaadin-combo-box label="Element" item-label-path="name" item-value-path="symbol" style="width: 240px;">
-          <template>
-            <b>[[item.name]], <sub>[[item.number]]</sub>[[item.symbol]]</b><br>
-            array index: [[index]]
-          </template>
-        </vaadin-combo-box>
-        <script>
-          window.addDemoReadyListener('#combo-box-styling-demos-custom-item-template', function(document) {
-            document.querySelector('vaadin-combo-box').items = elementsJson;
-          });
-        </script>
-      </template>
-    </vaadin-demo-snippet>
-
-
-    <h3>Configuring Icons</h3>
-    <vaadin-demo-snippet id="combo-box-styling-demos-configuring-icons" when-defined="vaadin-combo-box-light">
-      <template preserve-content>
-        <style>
-          paper-icon-button {
-            opacity: 0.4;
-            width: 24px;
-            height: 24px;
-            padding: 2px;
-          }
-        </style>
-        <vaadin-combo-box-light>
-          <vaadin-text-field label="Custom Icons">
-            <paper-icon-button slot="suffix" icon="remove-circle" class="clear-button"></paper-icon-button>
-            <paper-icon-button slot="suffix" icon="search" class="toggle-button"></paper-icon-button>
-          </vaadin-text-field>
-        </vaadin-combo-box-light>
-        <script>
-          window.addDemoReadyListener('#combo-box-styling-demos-configuring-icons', function(document) {
-            // NOTE: elements is an array of the form {'Carbon', ...}
-            document.querySelector('vaadin-combo-box-light').items = elements;
-          });
-        </script>
-      </template>
-    </vaadin-demo-snippet>
-
-
-    <h3>Replacing the Input</h3>
-    <p>
-      If you are using other custom input fields like &lt;iron-input>, you
-      need to define the name of the bindable property with the `attrForValue` attribute.
-    </p>
-    <vaadin-demo-snippet id="combo-box-styling-demos-replacing-the-input-with-an-iron-input" when-defined="vaadin-combo-box-light">
-      <template preserve-content>
-        <vaadin-combo-box-light attr-for-value="bind-value">
-          <iron-input>
-            <input>
-          </iron-input>
-        </vaadin-combo-box-light>
-        <script>
-          window.addDemoReadyListener('#combo-box-styling-demos-replacing-the-input-with-an-iron-input', function(document) {
-            document.querySelector('vaadin-combo-box-light').items = elements;
-          });
-        </script>
-      </template>
-    </vaadin-demo-snippet>
-
-
-    <h3>Using Focused and Selected Flags</h3>
-    <vaadin-demo-snippet id="combo-box-styling-demos-focused-and-selected-flags" when-defined="vaadin-combo-box">
+    <h3>Custom input field style</h3>
+    <vaadin-demo-snippet id="combo-box-border-style" when-defined="vaadin-combo-box">
```

Comment:
Rename to "input-field-style", the demo is titled like that and "border" is just one of the properties modified in the demo.

Reply:
Done